src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -625,14 +629,21 @@ public Response validateCondition(@Valid @NotNull Policy policy) {
             @APIResponse(responseCode = "500", description = "Internal error")
     })
     @Parameter(name = "id", description = "UUID of the policy")
-    public Response validateName(@NotNull JsonString policyName, @QueryParam("id") UUID id) {
+    public Response validateName(@NotNull String jsonPolicyName, @QueryParam("id") UUID id) {
```

Comment:
I changed a bit here this part. 
We are reciving a json string. If we want to read  `my cool policy` we expect to receive the raw string `"my cool policy"` which is a json string (application/json) as opposed to plain text.

For some reason after the change from json-b to jackson this wasn't working correctly. So I'm using the ObjectMapper below to parse the string (which should be a string).